com.ibm.wala.cast.python.test/data/tf2pp.py

```diff
@@ -0,0 +1,9 @@
+import tensorflow as tf
+
+@tf.function()
```

Comment:
How about one without the parentheses?

Reply:
The test without parenthesis is already there in `tf2t.py`.